strato_query/base_API_query.py

```diff
@@ -120,6 +115,40 @@ def join(self) -> Union[None, dict]:
     def order(self) -> Union[None, Tuple[str, ...]]:
         return self._order
 
+
+class APIMeanQueryParams(APIQueryParams):
+    def __init__(self, mean_variable_name: str, **kwargs):
+        assert mean_variable_name is None or isinstance(mean_variable_name, str)
```

Comment:
Can it be `None`? Doesn't it have to be a `str`?

Reply:
It does have to be a `str`. Thanks.